### PR TITLE
Feat/drop jss

### PR DESCRIPTION
**Pull request by prieston**

### CODE REVIEW

File: demo/index.js [hunk 1]
```diff
@@ -133,37 +133,37 @@ const progressBar = new MCGraphs.ProgressBar(
...
-  },
-  {
-    selector: "#htmlclip",
-    containerParams: {
-      width: "1024px",
-      height: "768px",
-    },
-  }
-);
+// todo: make the following work
```
Comment (alexdor) [inline demo/index.js:136]: is this not fixed?

File: src/Incidents/lineGraph/lineGraphStylesheet.js [hunk 2]
```diff
@@ -1,29 +1,24 @@
...
 
 export default function buildCSS(lineGraph) {
-  const createGenerateId = () => {
-    return (rule) => rule.key;
-  };
-  jss.setup({ createGenerateId });
-
   const styles = {
-    "container-lineGraph": {
+    "..container-lineGraph": {
```
Comment (alexdor) [inline src/Incidents/lineGraph/lineGraphStylesheet.js:6]: what are the `..` here?

File: demo/index.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import { loadPlugin, HTMLClip } from "@donkeyclip/motorcortex";
 import Player from "@donkeyclip/motorcortex-player";
-import MotorCortexGraph from "../dist/motorcortex-graphs.esm";
+import MotorCortexGraph from "../src";
```
Comment (alexdor) [inline demo/index.js:3]: this will cause webpack to compile src as well which means that it will create a different bundle than the production one, why are we doing this?

File: src/index.js [hunk 4]
```diff
@@ -4,8 +4,8 @@ import ProgressMeter from "./Incidents/progressMeter/progressMeter";
 import LineGraph from "./Incidents/lineGraph/lineGraph";
 import PieChart from "./Incidents/pieChart/pieChart";
 import validationRules from "./validation";
-import { name, version } from "../package.json";
-
+import conf from "../package.json";
+const { name, version } = conf;
```
Comment (alexdor) [inline src/index.js:8]: why this change?

File: src/shared/helpers.js [hunk 5]
```diff
@@ -18,3 +18,14 @@ export default {
...
   return typeof value === "number" && isFinite(value);
 }
+
+export function cssObjectToString(style) {
+  return Object.entries(style)
+    .map(
+      ([k, v]) =>
+        `${k}{${Object.entries(v)
+          .map(([k, v]) => `${k}:${v}`)
+          .join(";\n")}}`
```
Comment (alexdor) [inline src/shared/helpers.js:28]: Do we need a new line here? doesn't it just create additional bytes that aren't required?

File: src/shared/helpers.js [hunk 6]
```diff
@@ -18,3 +18,14 @@ export default {
...
+    )
+    .join("\n");
```
Comment (alexdor) [inline src/shared/helpers.js:30]: Do we need a new line here? doesn't it just create additional bytes that aren't required?

**Review by alexdor — CHANGES_REQUESTED** [review]

File: demo/index.js (hunk 1 above)
Comment (prieston) [reply demo/index.js:136]: nope this is the battery inc

File: demo/index.js (hunk 3 above)
Comment (prieston) [reply demo/index.js:3]: just wanted to check smthng, on it

File: src/Incidents/progressMeter/progressMeterStyleSheet.js [hunk 7]
```diff
@@ -83,15 +79,15 @@ export default function buildCSS(progressMeter) {
...
       "align-items": "center",
     },
-    "svg-preset": {
+    ".svg-preset": {
       width: `${progressMeter.boxSize * 0.5}px`,
       height: `${progressMeter.boxSize * 0.5}px`,
     },
-    "gradient-filter": {},
-    "gradient-stop": {
+    ".gradient-filter": {},
```
Comment (alexdor) [inline src/Incidents/progressMeter/progressMeterStyleSheet.js:86]:
```suggestion
```

**Review by alexdor — CHANGES_REQUESTED** [review]

File: demo/index.js [hunk 8]
```diff
@@ -5,7 +5,7 @@ import barChartData from "./data/barChartData.json";
 import lineGraphData from "./data/lineGraphData.json";
 import pieChartData from "./data/pieChartData.json";
 import progressBarData from "./data/progressBarData.json";
-import progressMeterData from "./data/progressMeterData.json";
+// import progressMeterData from "./data/progressMeterData.json";
```
Comment (alexdor) [inline demo/index.js:8]:
```suggestion
```

File: demo/index.js [hunk 9]
```diff
@@ -133,37 +133,37 @@ const progressBar = new MCGraphs.ProgressBar(
...
+//     },
+//   },
+//   {
+//     selector: "#htmlclip",
+//     containerParams: {
+//       width: "1024px",
+//       height: "768px",
+//     },
+//   }
+// );
```
Comment (alexdor) [inline demo/index.js:160]:
```suggestion
```

File: demo/index.js [hunk 10]
```diff
@@ -133,37 +133,37 @@ const progressBar = new MCGraphs.ProgressBar(
...
 
 clip.addIncident(barChart, 0);
 clip.addIncident(lineGraph, 5500);
 clip.addIncident(pieChart, 20500);
 clip.addIncident(progressBar, 26000);
-clip.addIncident(progressMeter, 31500);
+// clip.addIncident(progressMeter, 31500);
```
Comment (alexdor) [inline demo/index.js:166]:
```suggestion
```

**Review by alexdor — CHANGES_REQUESTED** [review]
